packages/coil-extension/src/popup/components/icons/CreditCardIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { SvgIconProps } from '@material-ui/core'
+
+import { AmericanExpress } from './AmericanExpress'
+import { DinersClub } from './DinersClub'
+import { Discover } from './Discover'
+import { JCB } from './JCB'
+import { MasterCard } from './MasterCard'
+import { Visa } from './Visa'
+import { CreditCardPlaceholder } from './CreditCardPlaceholder'
+
+//
+// Models
+//
+interface ICreditCardIcon extends SvgIconProps {
```

Comment:
Why not CreditCardIconProps ?

Reply:
I was actually surprised they got through, as actually @typescript-eslint used to ban that form (I used to use it too, though I adopted it from Java). It must be allowed these days by the default rules